src/embedded.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Embedded support for Flutter
```

Comment:
There are two potentially separate topics here:

1) Using Flutter on a "embedded device", typically a low-powered hardware device such as a smart-display, a thermostat, or similar

2) Embedding Flutter into a new operating system or environment, for example a new mobile platform or a new operating system

These two things are connected, but not necessarily the same, right?

Reply:
Correct. I can list both out but is there anything else we want to say about this?

Reply:
I think listing both of those scenarios is sufficient for now.